rocAL/rocAL/include/pipeline/master_graph.h

```diff
@@ -35,6 +35,10 @@ THE SOFTWARE.
 #include "node_fused_jpeg_crop_single_shard.h"
 #include "node_video_loader.h"
 #include "node_video_loader_single_shard.h"
+#include "node_audio_loader.h"
+#include "node_audio_loader_single_shard.h"
+#include "node_audio_loader.h"
+#include "node_audio_loader_single_shard.h"
```

Comment:
Why includes are added 2 times?

Reply:
I guess during some merge, this would have been changes. Will revert it

Reply:
reverted